_meta/config/cloudbeat.common.yml.tmpl

```diff
@@ -9,34 +9,6 @@ cloudbeat:
   type: {{$type}}
   # Defines how often an event is sent to the output
   period: 4h
-  fetchers:
-    - name: kube-api
-    - name: process
-      directory: "/hostfs"
-      processes:
-        etcd:
-        kube-apiserver:
-        kube-controller:
-        kube-scheduler:
-        kubelet:
-          config-file-arguments:
-            - config
-    - name: file-system
-      patterns: [
```

Comment:
Does this mean that these patterns are not configurable anymore?

Reply:
that's correct, it was already hard coded in the hbs file